extension/benchmark/README.md

```diff
@@ -0,0 +1,70 @@
+| InceptionV4 (ic4)     | XNN Q8       | [87.617](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218986716)               | [117.937](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218986716)                    | Samsung S22            |
+| InceptionV4 (ic4)     | QNN FP16     | [163.943](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218986381)              | [2.734](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218986381)                      | Samsung S22            |
+| InceptionV3 (ic3)     | XNN Q8       | [60.708](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218986023)               | [98.390](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218986023)                     | Samsung S22            |
+| InceptionV3 (ic3)     | QNN FP16     | [134.732](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218985425)              | [1.351](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218985425)                      | Samsung S22            |
+| TorchVisionVit (vit)  | QNN FP16     | [174.430](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218989581)              | [199.279](https://github.com/pytorch/executorch/actions/runs/10875550238/job/30218989581)                    | Samsung S22            |
+| ResNet50 (resnet50)   | COREML FP16  | [84.453](https://github.com/pytorch/executorch/actions/runs/10836589365/job/30078680731)               | [1329.846](https://github.com/pytorch/executorch/actions/runs/10836589365/job/30078680731)                   | Apple iPhone 15 Pro    |
+| Wave2Letter (w2l)     | XNN Q8       | [33.913](https://github.com/pytorch/executorch/actions/runs/10857890364/job/30139445319)               | [135.584](https://github.com/pytorch/executorch/actions/runs/10857890364/job/30139445319)                    | Apple iPhone 15 Pro    |
+
+
+## Supported Use Cases
+
+The benchmarking infrastructure currently supports two major use-cases:
+
+- **On-Demand Model Benchmarking:** Users can trigger benchmarking requests via GitHub Actions workflow dispatch UI. This feature will help backend developers collaborate with the ExecuTorch team to debug performance issues and advance state-of-the-art (SOTA) performance.
+
+- **Automated Nightly Batched Benchmarking:** The infrastructure performs automated nightly benchmarking to track and monitor performance over time. This allows for consistent performance monitoring and regression detection.
+
+
+## High-Level Diagram
+
+![Benchmarking Infrastructure](../../docs/source/_static/img/benchmark-infra.png)
+
+
+## Scheduling On-Demand Benchmarking
+
+Users can schedule a benchmarking workflow on a pull request through GitHub Actions using the workflow dispatch UI. Follow the steps below to trigger benchmarking:
```

Comment:
Not sure if we want to be clear here, only people with write permission to the repo can run the workflow.

Reply:
Will add it to the readme. Having write permission will limit the usage of it, especially for backend vendors. Except for explicitly grant write permissions to individuals, is there other more convenient way for vendors to start dogfooding?

Reply:
There are similar questions/requests to GH from what I see https://github.com/orgs/community/discussions/49688.  We could consider building something like `@pytorchbot` here, for example, posting a comment like `@executorchbot benchmark` to run perf benchmark.  There are pros and cons, but that's a familiar approach that PyTorch is using